humanmine.ipynb

```diff
@@ -0,0 +1,572 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "geographic-department",
+   "metadata": {},
+   "source": [
+    "# Genes expressed in pancreas in relation to diabetes\n",
```

Comment:
```suggestion
    "# Diabetes related genes expressed in pancreas\n",
```

Reply:
This should address the comment of @joshmoore - I am almost certain we will not figure out what exactly the "relation" is, as this is probably "diabetes is mentioned in the annotation".

Reply:
The problem we are having here is that the formulation "Genes expressed in pancreas in relation to diabetes" is used in other places of the notebook and is hard-coded in the newly added image too. 
I think that "Genes expressed in pancreas in relation to diabetes" suggests that the genes in question are expressed only when triggered by diabetes, and this is most probably not true if I look at the humanmine query in the notebook. It will rather be that these genes are somehow influenced/their expression modified by diabetes, which is a much softer requirement returning more genes I imagine.

Reply:
We can redo the image This is not a problem.

Reply:
updated